persist/src/main/resources/db/migration/V5__AuditTables.sql

```diff
@@ -0,0 +1,7 @@
+CREATE TABLE IF NOT EXISTS audit (
+    transaction_type VARCHAR(1000) NOT NULL,
+    table_name       VARCHAR(1000) NOT NULL,
+    user_id          INTEGER NOT NULL,
+    result           VARCHAR (1000) NOT NULL,
```

Comment:
What does `result` represent?

Reply:
Result is to represent the changed the row so like on UPDATE the result column would be the updated row